Makefile

```diff
@@ -61,11 +61,16 @@ ShellyU: MOS_BUILD_FLAGS=--build-var=ASAN=1 --build-var=UBSAN=1
 ShellyU: build-ShellyU
 	@true
 
-fs/index.html.gz: fs_src/index.html fs_src/style.css fs_src/script.js fs_src/logo.svg
-	sed -e '/<style>/ r fs_src/style.css' \
-		-e '/<script>/ r fs_src/script.js' \
-		-e '/<!-- logo.svg .* -->/ r fs_src/logo.svg' \
-		fs_src/index.html 2>&1 | gzip -9 -c > fs/index.html.gz
+fs/index.html.gz:
+	rm -f fs/index.html.gzip && \
```

Comment:
Without this Make often tells me that no change is needed.

Reply:
dependency updates are core to make functionality, we should not be rebuilding things that haven't changed.
this never happened to me, incremental builds are quite reliable. please keep the dependencies for now.

Reply:
It always happens if you switch between branches and the date of a source file is younger than the generated file.
But I will revert that change and investigate it more and create a new issue/pr.

Reply:
here is a case:
```bash
➜  shelly-homekit git:(master) g checkout ui-confirm
Switched to branch 'ui-confirm'
Your branch is up to date with 'my/ui-confirm'.
➜  shelly-homekit git:(ui-confirm) g merge master
warning: Cannot merge binary files: fs/index.html.gz (HEAD vs. master)
Auto-merging fs_src/script.js
Removing fs/logo.png
Auto-merging fs/index.html.gz
CONFLICT (content): Merge conflict in fs/index.html.gz
Automatic merge failed; fix conflicts and then commit the result.
➜  shelly-homekit git:(ui-confirm) ✗ make fs/index.html.gz
make: `fs/index.html.gz' is up to date.
➜  shelly-homekit git:(ui-confirm) ✗
```